source/material_model/rheology/elasticity.cc

```diff
@@ -474,7 +773,17 @@ namespace aspect
       Elasticity<dim>::
       calculate_elastic_viscosity (const double shear_modulus) const
       {
-        return shear_modulus*elastic_timestep() + elastic_damper_viscosity;
+        // In case the computational timestep dtc differs from the elastic timestep,
+        // we need to scale the elastic viscosity with the timestep ratio dtc/dte:
+        // $\eta_{el} = \Delta t_{el} G$
+        // $\eta_{el}^{c} = \Delta t_{el} G \frac{\Delta t_c}{\Delta t_{el}} = \Delta t_c G$.
+        // Since we already have a function that returns the timestep ratio $\frac{\Delta t_c}{\Delta t_{el}}$,
+        // use that instead.
+        // TODO: what to do with the damper? It is asserted to be zero when using
+        // this rheology in the visco_plastic and viscoelastic material models. But should
+        // the scaling apply to the dampened viscosity or the undampened viscosity?
```

Comment:
I think this is a question about which value the user should input, the damper viscosity over the computational timestep `elastic_damper_viscosity_c` or over the elastic timestep `elastic_damper_viscosity_e`. I think it makes most sense for the user to input `elastic_damper_viscosity_c` over the computational timestep. This will be more "physical", not produce odd behaviour if the elastic timestep changes, and simulate a "Voigt" element with an isoviscous damper). 

Over the elastic timestep, we have `eta_el_e = shear_modulus*elastic_timestep() + elastic_damper_viscosity_e`
Also, from the derivation in our paper:
<img width="96" alt="image" src="https://github.com/geodynamics/aspect/assets/8144905/b5835c7b-134c-4a0d-bb5b-c93adbc8294a">

<img width="86" alt="image" src="https://github.com/geodynamics/aspect/assets/8144905/32b6fad4-22a0-4079-92db-16474d8db1f8">

So this implies that:
`shear_modulus*elastic_timestep()*timestep_ratio + elastic_damper_viscosity`
which is what you have in the code already.

Reply:
Great, thanks!